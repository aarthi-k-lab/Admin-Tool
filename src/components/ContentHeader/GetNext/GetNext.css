.get-next.get-next {
  background-color: var(--light-blue-900);
  margin-right: 0.5rem;
}

.get-next.get-next:hover {
  background-color: var(--blue-900);
}
