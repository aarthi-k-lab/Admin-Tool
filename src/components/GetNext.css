.getnext {
  background-color: rgb(248, 250, 250);
  min-height: 55px;
  margin: -8px;
  margin-left: 52px;
  margin-top: 10px;
  width: 100vw;
  position: absolute;
}

.getnext-h1 {
  font-family: 'Lato', sans-serif;
  font-weight: 400;
  position: absolute;
  top: 14px;
  margin-left: 15px;
}
