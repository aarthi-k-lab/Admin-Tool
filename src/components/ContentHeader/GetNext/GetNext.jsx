import React from 'react';
import PropTypes from 'prop-types';
import Button from '@material-ui/core/Button';
import './GetNext.css';

const GetNext = ({ disabled, onClick }) => (
  <Button
    className="material-ui-button"
    color="primary"
    disabled={disabled}
    onClick={onClick}
    styleName={disabled ? 'get-next-disabled' : 'get-next'}
    variant="contained"
  >
    Get Next
  </Button>
);

GetNext.defaultProps = {
  disabled: false,
  onClick: () => {},
};

GetNext.propTypes = {
  disabled: PropTypes.bool,
  onClick: PropTypes.func,
};

export default GetNext;
