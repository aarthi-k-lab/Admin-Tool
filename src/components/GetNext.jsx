import React from 'react';

import './GetNext.css';

function GetNext() {
  return (
    <div styleName="getnext">
      <div styleName="getnext-h1"> Document Verification </div>
    </div>
  );
}

export default GetNext;
