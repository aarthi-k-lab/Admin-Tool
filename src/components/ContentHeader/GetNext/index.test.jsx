import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import GetNext from '.';

describe('<GetNext />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<GetNext />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('triggers onClick handler', () => {
    const handleClick = jest.fn();
    const wrapper = shallow(<GetNext onClick={handleClick} />);
    wrapper.simulate('click');
    expect(handleClick.mock.calls.length).toBe(1);
  });
});
