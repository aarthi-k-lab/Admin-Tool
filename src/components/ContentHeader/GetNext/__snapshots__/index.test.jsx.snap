// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetNext /> snapshot test 1`] = `
<button
  className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedPrimary-13 MuiButton-raised-15 MuiButton-raisedPrimary-16 material-ui-button components-ContentHeader-GetNext-GetNext__get-next--3nV4r"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    Get Next
  </span>
  <span
    className="MuiTouchRipple-root-30"
  />
</button>
`;
